src/xo-app/backup/restore/index.js

```diff
@@ -98,16 +102,53 @@ const doImport = ({ backup, sr, start }) => {
   }
 }
 
+class _Collapsible extends Component {
+  render () {
+    const {
+      collapsible,
+      children,
+      ...props
+    } = this.props
+
+    return collapsible
+      ? <Collapse {...props}> {children} </Collapse>
+      : <div> {children} </div>
+  }
+}
+
 class _ModalBody extends Component {
   get value () {
     return this.state
   }
 
+  _srPredicate = createSelector(
```

Comment:
This should be a `_getSrPredicate` and return a predicate function instead of being a predicate itself because everytime you'll call it with a different SR, the selector will trigger an execution of the callback since the SR will be different from the one in its cache.

@julien-f Can you confirm this?